src/main/resources/reference.conf

```diff
@@ -515,6 +515,10 @@ steps: {
       }
       targets: ${steps.target.output.target}
       mechanism_of_action: ${steps.drug.output.mechanism_of_action}
+      literature_dating_input: {
+        format: "json"
+        path: "gs://otar000-evidence_input/literature_export/*"
```

Comment:
This is temporary. Before merging we need to sort out the proper integration. It should be there for the test run.